pkgs/tools/wayland/gtklock/default.nix

```diff
@@ -51,8 +49,8 @@ stdenv.mkDerivation rec {
       Important note: for gtklock to work you need to set "security.pam.services.gtklock = {};" manually.
     ''; # Following  nixpkgs/pkgs/applications/window-managers/sway/lock.nix
     homepage = "https://github.com/jovanlanik/gtklock";
-    license = licenses.gpl3;
-    maintainers = with maintainers; [ dit7ya ];
+    license = licenses.gpl3Only;
```

Comment:
Is this the same reasoning as above?

Reply:
The license is mentioned as `GPLv3` without approving above versions, see https://github.com/jovanlanik/gtklock/blob/48001a2ca5553edf5a2f3390866d37fbc4b95067/meson.build#L7